zebra/zebra_dplane.h

```diff
@@ -178,6 +178,10 @@ enum dplane_op_e {
 
 	DPLANE_OP_NEIGH_TABLE_UPDATE,
 	DPLANE_OP_GRE_SET,
+
```

Comment:
semantic used does not explicitly say this message is for incoming operation or outgoing operation.
should we have to split the lists in two or not ?

Reply:
I don't think the direction really matters, does it - the handlers of the various operation codes know what to do?